common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -404,6 +404,35 @@ def test_video_with_youtube_blocked(self):
 
         self.assertTrue(self.video.is_video_rendered('html5'))
 
+    def test_html5_video_caption_fetch_with_youtube_id(self):
+        """
+        Scenario: If there are no transcripts for HTML5 mode and there exist
+        a transcript for Youtube mode with YouTubeId then fetch the transcript with
+        for HTML5 mode with YouTubeId
+        Given the I have uploaded a .srt.sjson file to assets for Yotube mode
+        And the YouTube API is blocked
+        And the course has a Video component in "Youtube_HTML5" mode
+        And Video component rendered in "HTML5" mode
+        And Html5 mode video have no transcripts
+        Then the HTML5 video mode fetch the caption using YouTubeId
+        """
```

Comment:
The doc string have many grammatical & spelling mistakes. This could be like:

Scenario: User should see Youtube captions for HTML5 video If there are no transcripts available for HTML5 mode
Given that I have uploaded a .srt.sjson file to assets for Youtube mode
And the YouTube API is blocked
And the course has a Video component in "Youtube_HTML5" mode
And Video component rendered in HTML5 mode
And Html5 mode video have no transcripts
When i see the captions for HTML5 mode video
Then i should see the Youtube captions